### DISCUSSION TITLE
原生的duilib的虚拟窗口是否在网易的duilib里面有替代或者相似的功能？

### DISCUSSION THREAD

**Discussion by QHHNB** [netease-im/NIM_Duilib_Framework#419] [General]
之前项目是用原生的duilib开发的，现在准备换成网易的，原项目为了避免所有消息响应和xml都写在一个文件里，用了原生duilib的虚拟窗口的技术，现在换了网易的找了半天没找到，不知道网易duilib是否有替代或者类似？

**Comment by nmgwddj** [comment]
您指的虚拟窗口是虚列表还是？能否更详细的描述一下您的需求，或上传一份示例代码。
